### ISSUE TITLE
RChain Chinese Community First  Meetup

### ISSUE THREAD

**Issue by wangjia184** [rchain/bounties#724]
* Lecture to propagate RChain and explain why it is revolutionary by comparing its scalability / concurrency / compositionality with tranditional von Neumann style computering.
* Lecture to propagate RChain's advantages/friendliness on business
* To enhance the influence of RChain 

**Time**: 2018-05-20 15:00 UTC+8 (BeijingTime)  
**Location** :  Room 313, Building on Water ,Hunan University, Changsha, HuNan, China

**Attendees** : 
About 40-50 attendees from RChain Chinese Community attended the meeting. Also there was a live video broadcast and about 300 persons from Chinese official QQ group 492114454 watched it.

**Media**:
* Live Video broadcasting
* Chinese Social Media including WeChat / WebBo
* https://mp.weixin.qq.com/s/yMuNQQC530dBp0_D6eSuEw
* https://mp.weixin.qq.com/s/UZi3gnIzZemlbaI7vW6EEA

**Agenda**:

1.  The vision of RChain 
     -- From business perspective to explain RChain's advantages
     Speaker : Eric.Ren CTO of Fintech Research Center of Hunan University, RChain verified member

2.  Revolution of Computering :  RChain Introduction
     -- Explain how RChain works and how it is distinct from von Neumann style computering.
     Speaker : Jerry.Wang  Software Architect with about 20 years experience on development, RChain verified member
     Video : https://www.youtube.com/watch?v=wxqOlYXny7c
     Slides : https://docs.google.com/presentation/d/122Y6eVp2c_pXT_ztRkjvyJgXsSR7WJ5jT15K9DGWtko/edit

     The lecture is made up by 6 parts.
  
     1. Theory & Background
          * Review the development history of von Neumann history and figure out its bottleneck
          * 4C and their meaning
          * Development history of RhoCalculus
     2. Rho-Calculus
          * Explain the world view of Rho-Calculus
          * Introduce Rho-Calculus terms.
          * Explain each term with example and analogy (e.g. name as mailbox)
     3. Rholang and its operating mechanism
          * What a contract is.
          * Explain the state change resulted by rholang code execution with analogy . (e.g tuplespace as mailboxes-cabinet)
     4. Compositionality
          * Explain why RChain owns this distinct feature with analogy (e.g tuplespace as mailboxes-cabinet)
          * Explain what a region is
     5. Namespace
          * Explain consensus and why namespace is needed.
          * Explain why the set of namespace is the powerset of regions
          * Explain namespace is a set of names
     6. Other
          * Correct-by-Construction
          * LADL
          * blockDAG

3.  RChain and its Competitor
     -- Review RChain by comparing with other competitors from multiple perspectives. 
     Speaker : Tao.Huang  Blockchain Researcher of bixin.com, RChain verified member

4. Free Talk
    Speaker : Fans of RChain

**Effort & Cost**

1. Advertisement & Organization 
2. Meeting Room Rent Fee
3. Posters, Banners
4. Fruits, Water and Dinner
5. Video Streaming / Capture / Post-Production
6. Every speaker spent several days to prepare the slides and lecture.

_Budget of Task:_ **$4000**  \
_Timeline Required to Complete the Task:_ **3 days**  \
_Measure of Completion:_  **We successfully carries out this meetup**.

|  Person         | Discord ID    | GitHub ID     | What she/he did       | Percentage         |
|---------------|---------------|---------------|------------------------|-------------------|
|  Jerry.Wang  |  Jerry.Wang#2836  |  wangjia184  |  Lecture                     |   25%                  |
|  Eric.Ren       |   Eric.Ren#5921  |   @cmrc2018 |  Organizer /  Lecture |   25%                  |
|  Tao.Huang   |  TaoHuang#2777   |  @taobixin   |  Organizer / Lecture  |   25%                  |
|  Tiff.Wang    |    Tiff.Wang#9835  |   @TiffWang     |  Device Support/Video Record and Post-production  |   13%                  |
|  MingYan.Chen|  CaroChan#9107   |  @cmycaro  |  ClassRoom&Restaurant Booking / Emcee / Misc  |   12%                  |

--  \
***Legal***  \
_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by zsluedem** [comment]
@wangjia184 I think you have to [become the collaborators](https://docs.google.com/forms/d/e/1FAIpQLSdP42ZJ-UQ8HU1QkO_lywWbfAqxTJWYG2TWRgbQon6h5BeuKw/viewform) first~!
Listing others' members who have contributed to the meetup would be more helpful for others to vote for the budget.

BTW, it is really a good first meetup in China.

**Comment by Ojimadu** [comment]
@wangjia184 It seems you're not yet a collaborator. Follow the instructions above to become one, or if any of the participants you mentioned is already a member and a collaborator, s/he can help claim the rewards for subsequent redistribution to other participants.

**Comment by Ojimadu** [comment]
I have voted for B&R on this issue but I could not find Caro and Tao  to set rewards for them. This needs one more vote for budgets and two more votes for rewards.

**Comment by wangjia184** [comment]
Thanks @Ojimadu for voting, we are new here hence not familiar with the flows. I am trying hard to follow :)

I have 
* filled the RAM-registration form
* mailed ops@rchain.coop with my Discord ID / GitHubID
* Joined RChain Google Group


>if any of the participants you mentioned is already a member and a collaborator, s/he can help claim the rewards for subsequent redistribution to other participants.

If I claim the reward and redistribute to them, are their names still required to appear in rewards.rchain.coop ?

**Comment by Ojimadu** [comment]
Your name would be the one to appear on the B&R as you received the reward on their behalf.
On the RAM registrations, I supposed you should receive an invitation to become a collaborator on this repository. Have you been KYC'ed on Discord?

**Comment by wangjia184** [comment]
Yep, I have been KYC'ed on Discord.
And I filled both [collaborators form](https://docs.google.com/forms/d/e/1FAIpQLSdP42ZJ-UQ8HU1QkO_lywWbfAqxTJWYG2TWRgbQon6h5BeuKw/viewform) and [RAM-registration](https://docs.google.com/forms/d/e/1FAIpQLSecwGUVFNx_Xa_Qsw5bxLnaKstPS8kQnfrUGqpuf22rLDteDg/viewform) form, no invitation email is received so far.  Perhaps I need wait longer?

**Comment by wangjia184** [comment]
Hi @Ojimadu 

I(@wangjia184) and @TiffWang received the rewards.

However,  @cmrc2018, @taobixin, @cmycaro they havent receive the rewards last month because they registered late and dont have insufficient votes.

Could you please kindly vote for them in this month?

https://rewards.rchain.coop/index.php?-table=issue&-search=China&-action=view&-submit=Search+Issue&-sort=num+desc&-cursor=7&-skip=0&-limit=30&-mode=list&-recordid=issue%3Fnum%3D724

Thank you

**Comment by Ojimadu** [comment]
Done.

**Comment by Ojimadu** [comment]
@zsluedem This issue needs one vote to be complete in budgets and reward respectively. Would like to get this closed once the votes are complete.

**Comment by zsluedem** [comment]
@Ojimadu OK~! I have asked @sportshark to vote for the budget~! I think it's all done now~!

**Comment by sportshark** [comment]
@Ojimadu done!

**Comment by allancto** [comment]
Hi, it appears there are rewards due to @carochan @taobixin, but you need to first please join (see contributing.md) and then you will be able to submit a declaration with invoice. If you have any questions please contact myself or @wangjia184 or @Ojimadu. 

Thanks! -@allancto

**Comment by cmrc2018** [comment]
@allancto why  haven't I get the rewards so far? what's the reason?

**Comment by Ojimadu** [comment]
@cmrc2018 Have you submitted an invoice?

**Comment by Ojimadu** [comment]
The invoice submitted was for which month?

**Comment by wangjia184** [comment]
@Ojimadu do you find their invoices :) ?

**Comment by Ojimadu** [comment]
@cmrc2018 Please remove the link to your invoice. 
Everything looks fine with the invoice, have you filled out the tax form?

cc: @allancto can you have a look at this?

**Comment by Carochan** [comment]
I am not chinese. thanks



2018-08-18 13:38 GMT+02:00 cmrc2018 <notifications@github.com>:

**Comment by cmrc2018** [comment]
@Ojimadu I have filled the tax form already

**Comment by Ojimadu** [comment]
You are not the only one having such a problem. I don't know why but I would check.
